content/en/docs/reference/using-api/health-checks.md

```diff
@@ -27,15 +27,11 @@ The following examples will show how you can interact with the health API endpoi
 For all endpoints you can use the `verbose` parameter to print out the checks and their status.
 This can be useful for a human operator to debug the current status of the Api server, it is not intended to be consumed by a machine:
 
-    ```shell
```

Comment:
Hi @jjasghar .
Instead, could you remove the indentation (and leave the code block)?
I suspect the formatting will be corrected.

Reply:
updated and amended :)